### PR TITLE
[#828] Updated Setup to 4.9 final. introduced 4.10-ibuilds

### PR DESCRIPTION
**Pull request by cdietrich**
[#828] Updated Setup to 4.9 final. introduced 4.10-ibuilds
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: releng/org.eclipse.xtext.contributor/Xtext.setup [hunk 1]
```diff
@@ -884,7 +887,40 @@
           <repository
               url="${p2.draw2d}"/>
           <repository
-              url="http://download.eclipse.org/eclipse/updates/4.9-I-builds"/>
+              url="http://download.eclipse.org/eclipse/updates/4.9-I"/>
```
Comment (kthoms) [inline releng/org.eclipse.xtext.contributor/Xtext.setup]: remove

**Comment by kthoms** [comment]
Modular target "Photon - Integration" can be removed

File: releng/org.eclipse.xtext.contributor/Xtext.setup (hunk 1 above)
Comment (kthoms) [reply releng/org.eclipse.xtext.contributor/Xtext.setup]: The additional eclipse/updates repository is only needed when the base platform target should be overridden. The Eclipse Platform already defines the typical main repos for each release. Only for the upcoming release we need the I-build repo additionally.

**Comment by cdietrich** [comment]
updated the pr @kthoms

**Pull request merged**